ddtrace/contrib/flask/middleware.py

```diff
@@ -158,6 +158,9 @@ def _finish_span(self, span, exception=None):
             method = request.method
             endpoint = request.endpoint or code
             url = request.base_url or ''
+            query_string = compat.to_unicode(request.query_string)
```

Comment:
This Flask middleware is actually deprecated, nobody should be using this. You'll want to update `contrib/flask/path.py`.

https://github.com/DataDog/dd-trace-py/blob/4313f388383b90ccf2bcbca9d7ef1c400c827ece/ddtrace/contrib/flask/patch.py#L330